### PR TITLE
chore: improve MissingTripEdgeValidator code clarity

### PR DESCRIPTION
**Pull request by lionel-nj**
**Summary:**

This PR: 
- removes a useless statement
- uses `for` loop instead of lambda to improve code clarity

**Expected behavior:** 

Same tests should pass.

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Run the unit tests with `gradle test` to make sure you didn't break anything
- [x] Format the title like "feat: [new feature short description]". Title must follow the Conventional Commit Specification(https://www.conventionalcommits.org/en/v1.0.0/).
- [x] Linked all relevant issues
- ~[ ] Include screenshot(s) showing how this pull request works and fixes the issue(s)~

### CODE REVIEW

**Review by barbeau — APPROVED** [review]
👍

File: main/src/main/java/org/mobilitydata/gtfsvalidator/validator/MissingTripEdgeValidator.java [hunk 1]
```diff
@@ -36,46 +38,44 @@
...
+      List<GtfsStopTime> stopTimesForTrip = entry.getValue();
+      GtfsStopTime tripFirstStop = stopTimesForTrip.get(0);
+      GtfsStopTime tripLastStop = stopTimesForTrip.get(stopTimesForTrip.size() - 1);
+      if (!tripFirstStop.hasArrivalTime()) {
+        noticeContainer.addValidationNotice(
+            new MissingTripEdgeNotice(
+                tripFirstStop.csvRowNumber(),
+                tripFirstStop.stopSequence(),
+                tripId,
+                "arrival_time"));
```
Comment (aababilov) [inline main/src/main/java/org/mobilitydata/gtfsvalidator/validator/MissingTripEdgeValidator.java:53]: Here and below: please use GtfsStopTimeTableLoader.ARRIVAL_TIME_FIELD_NAME. This way you won't risk making a typo in the field name.

File: main/src/main/java/org/mobilitydata/gtfsvalidator/validator/MissingTripEdgeValidator.java [hunk 2]
```diff
@@ -36,46 +38,44 @@
...
+      }
+      if (!tripFirstStop.hasDepartureTime()) {
+        noticeContainer.addValidationNotice(
+            new MissingTripEdgeNotice(
+                tripFirstStop.csvRowNumber(),
+                tripFirstStop.stopSequence(),
+                tripId,
+                "departure_time"));
```
Comment (aababilov) [inline main/src/main/java/org/mobilitydata/gtfsvalidator/validator/MissingTripEdgeValidator.java:61]: DEPARTURE_TIME_FIELD_NAME here and below.

You can import both static fields:

import static org.mobilitydata.gtfsvalidator.table.GtfsStopTimeTableLoader.DEPARTURE_TIME_FIELD_NAME;

**Comment by aababilov** [comment]
Thank you for doing this!

File: main/src/main/java/org/mobilitydata/gtfsvalidator/validator/MissingTripEdgeValidator.java (hunk 1 above)
Comment (lionel-nj) [reply main/src/main/java/org/mobilitydata/gtfsvalidator/validator/MissingTripEdgeValidator.java:53]: True! Thank you @aababilov!

**Review by aababilov — APPROVED** [review]

**Comment by aababilov** [comment]
Ship it!

**Comment by lionel-nj** [comment]
Thanks for your review @aababilov  :)

**Comment by aababilov** [comment]
Thanks for the nice code @lionel-nj !

**Pull request merged**